src/main/app.js

```diff
@@ -59,37 +68,96 @@ fluid.defaults("gpii.app", {
             activeSet: null
         }
     },
-    // prerequisites
+    members: {
+        machineId: "@expand:{that}.installID.getMachineID()"
+    },
     components: {
-        psp: {
-            type: "gpii.app.psp",
-            createOnEvent: "onPrerequisitesReady",
+        networkCheck: { // Network check component to meet GPII-2349
```

Comment:
This component can be removed since it was only required for the HST testing deployment. It got into the main branch through a faulty merging of build branches.